src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoCondition } from '../../types/TodoCondition';
+import { noop } from '../../utils/noop';
+
+type Props = {
+  todo: Todo,
+  todoCondition: TodoCondition,
+  onDeleteTodo?: (todoId: number) => void,
+  toggleTodo?: (curentTodo: Todo[], status?: boolean | undefined) => void,
```

Comment:
according to the parameter name it should be a single todo
`curentTodo: Todo[]`